include/RAJA/policy/cuda/kernel/For.hpp

```diff
@@ -162,12 +162,11 @@ struct CudaStatementExecutor<
 template <typename Data,
           camp::idx_t ArgumentId,
           int ThreadDim,
-          int MinThreads,
           typename... EnclosedStmts,
           typename Types>
 struct CudaStatementExecutor<
     Data,
-    statement::For<ArgumentId, RAJA::cuda_thread_xyz_loop<ThreadDim, MinThreads>, EnclosedStmts...>,
+    statement::For<ArgumentId, RAJA::cuda_thread_xyz_loop<ThreadDim>, EnclosedStmts...>,
```

Comment:
Why did MinThreads go away?

Reply:
It looks like because we had a change of definition of the following struct: 

```
template<int dim, int min_threads>
struct cuda_thread_xyz_loop{};
```
to 
```
template<int ... dim>                                                                                                                                                                                         
struct cuda_block_xyz_loop{}; 
```

What do you think @ajkunen , should we have separate policies for team loops? This is a good discussion point.

Reply:
Resolved, this change is okay.